seccomp_internal.go

```diff
@@ -14,7 +14,8 @@ import (
 // Get the seccomp header in scope
 // Need stdlib.h for free() on cstrings
 
-// #cgo pkg-config: libseccomp
+// #cgo CFLAGS: -I../seccomp/include
+// #cgo LDFLAGS: -L../seccomp/src/.libs -l:libseccomp.a
```

Comment:
same here, let's use pkg-config

Reply:
Agree.

It's possible to test `libseccomp-golang` against `libseccomp-2.5.0`, even if it is not installed, with the following:

```
cd ../libseccomp && mkdir -p prefix
./configure --prefix=$PWD/prefix && make && make install
cd ../libseccomp-golang
PKG_CONFIG_PATH=$PWD/../libseccomp/prefix/lib/pkgconfig/ make
LD_PRELOAD=$PWD/../libseccomp/prefix/lib/libseccomp.so.2.5.0 PKG_CONFIG_PATH=$PWD/../libseccomp/prefix/lib/pkgconfig/ make test
```